handler-simple/src/main/java/com/github/philippheuer/events4j/simple/domain/SimpleDisposableWrapper.java

```diff
@@ -46,7 +46,7 @@ public void dispose() {
             disposable.dispose();
 
             // remove from active subscriptions
-            activeSubscriptions.remove(this);
+            activeSubscriptions.remove(id);
```

Comment:
should there be a isDisposed check here?

Reply:
Only the EventManager / the disposable wrappers have write access to `activeSubscriptions`, therefore i think it's enough to check `.isDisposed()` of the disposable.

The subscription is only removed if `.dispose()` is called successfully.

Reply:
Nevermind, I was only looking at the diff & missed that the check is already present https://github.com/PhilippHeuer/events4j/blob/b0c6f9885ea76d1c503ae232094f631988eb3b25/handler-simple/src/main/java/com/github/philippheuer/events4j/simple/domain/SimpleDisposableWrapper.java#L44